src/test/unittest/valgrind.py

```diff
@@ -286,15 +288,15 @@ def verify(self):
                               .format(self.tool_name))
 
 
-def require_valgrind_enabled(valgrind):
+def require_valgrind_enabled(valgrind, check_log=True):
```

Comment:
Is there a way to make this a bit more verbose? I have no idea what `check_log` means in this context (I checked, but...).

Maybe let's call this `verify_no_errors` ?